range_pool.go

```diff
@@ -117,19 +121,24 @@ func (s *Service) Create(ctx context.Context, namespace, ID string, num, min, ma
 	// Fetch the latest item used.
 	var latest int
 	{
-		l, err := s.storage.Search(ctx, fmt.Sprintf(LatestKeyFormat, namespace))
+		k, err := microstorage.NewK(fmt.Sprintf(LatestKeyFormat, namespace))
```

Comment:
Looks like `microstorage.NewKF` could be a thing.

Reply:
Probably rather `microstorage.NewKf`.

Reply:
Uncertain. I'd rather prefer making this: `fmt.Sprintf(LatestKeyFormat` a function.